starboard/nplb/media_can_play_mime_and_key_system_test.cc

```diff
@@ -220,53 +239,59 @@ TEST(SbMediaCanPlayMimeAndKeySystem, Invalid) {
 }
 
 TEST(SbMediaCanPlayMimeAndKeySystem, MinimumSupport) {
-  // H.264 High Profile Level 4.2
-  SbMediaSupportType result = SbMediaCanPlayMimeAndKeySystem(
-      "video/mp4; codecs=\"avc1.64002a\"; width=1920; height=1080; "
-      "framerate=30; bitrate=20000000",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
-
-  // H.264 Main Profile Level 4.2
-  result = SbMediaCanPlayMimeAndKeySystem(
+  std::vector<const char*> params_fhd{
       "video/mp4; codecs=\"avc1.4d002a\"; width=1920; height=1080; "
-      "framerate=30;",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
-
-  result = SbMediaCanPlayMimeAndKeySystem(
-      "video/mp4; codecs=\"avc1.4d002a\"; width=0; height=0; "
-      "framerate=0; bitrate=0",
-      "");
-  ASSERT_EQ(result, kSbMediaSupportTypeProbably);
+      "framerate=30",
+      "video/mp4; codecs=\"avc1.64002a\"; width=1920; height=1080; "
+      "framerate=30",
```

Comment:
We can, but I think it's covered by the VerifyMaxBitrate test. Actually, there's a lot of overlap between these two tests, maybe we should just merge the two.

With the switch statement, all fhd mimes should be run with FHD devices or higher.

Reply:
On second thought, I see in your below comment that we should only run one set of params for each device type, rather than all params for that device type and below (ex. 4k devices run 4k and fhd params). I'll make the change to add avc queries to each of them